exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRoot.java

```diff
@@ -55,44 +59,89 @@ public Viewable getStats() {
   @GET
   @Path("/stats.json")
   @Produces(MediaType.APPLICATION_JSON)
-  public List<Stat> getStatsJSON() {
-    List<Stat> stats = Lists.newLinkedList();
-    stats.add(new Stat("Number of Drill Bits", work.getContext().getBits().size()));
-    int number = 0;
-    for (CoordinationProtos.DrillbitEndpoint bit : work.getContext().getBits()) {
-      String initialized = bit.isInitialized() ? " initialized" : " not initialized";
-      stats.add(new Stat("Bit #" + number, bit.getAddress() + initialized));
-      ++number;
+  public Stats getStatsJSON() {
+    final String version = work.getContext().getOptionManager().getOption(ExecConstants.CLUSTER_VERSION).string_val;
+
+    final Map<String, Object> props = Maps.newLinkedHashMap();
+    props.put("Cluster Version", version);
+    props.put("Number of Drillbits", work.getContext().getBits().size());
```

Comment:
"Number of Drillbits" looks like a user-visible string. Not clear how these properties are used. In general, property keys should be internal. Then, if this is for the UI, the UI template should contain the user-visible label. We generally do not want user-visible strings to appear in the body of Java code.

Seems this is for the REST API. So the properties appear as JSON properties? Best to use names that are valid identifiers: "clusterVersion", "drillbitCount", etc.